data/TaskRow.css

```diff
@@ -48,6 +48,11 @@ row.card {
     padding-top: 6px;
 }
 
+row check {
+    padding: 1px;
+    border-radius: 12px;
+}
```

Comment:
I'd prefer not to do this since we use the rounded shape elsewhere to indicate radio buttons

Reply:
Personally I think it looks much nicer, especially together with the potential spinner of #104 - but don't have too strong feelings about this. Removed the css style.